File: Desafios em C#/1020 - Idade em Dias.cs
using System; 

//Developed by: @LucasMarcuzo
class URI1020 {

    static void Main(string[] args) { 
        
        int notes, aux;

        notes=int.Parse(Console.ReadLine());

        Console.WriteLine("{0} ano(s)", notes/365);
        aux = (notes%365);

        Console.WriteLine("{0} mes(es)", aux/30);
        aux = (aux%30);

        Console.WriteLine("{0} dia(s)", aux/1);
   
   }
}

File: Desafios em JavaScript/1020 - Idade em Dias.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var notes, aux;

notes=parseInt(lines.shift());

var na = parseInt(notes/365);
console.log(`${na} ano(s)`);
aux = (notes%365);

var nm = parseInt(aux/30);
console.log(`${nm} mes(es)`);
aux = (aux%30);

var nd = parseInt(aux/1);
console.log(`${nd} dia(s)`);